### DISCUSSION TITLE
Trader Review section for secondary reputation metric

### DISCUSSION THREAD

**Discussion by RequestPrivacy** [bisq-network/bisq2#1870] [Ideas]
Users can already see the profiles of other users e.g. like this:

![image](https://github.com/bisq-network/bisq2/assets/72392544/0fd25697-e31d-49f9-9b58-922309427c94)

If bisq 2 would allow to add reviews to this user profile this could help lower reputation traders boost their own reputation, depending on how many high reputation traders reviewed/commented in a favorable way on his/her profile.

To prevent scammers from flooding the review section a minimum amount of reputation could be enforced (e.g. at least 1 star) and only possible before closing a trade.

So for people wondering if they should take a trade with a low reputation maker, they could go to his profile section and check profile age and profile reviews. If there are reviews from high reputation traders (e.g. 1x with 35.000 and another one with 50.000) then they could deduct by the reputation of others that the reviews and the peer is legit

**Comment by HenrikJannsen** [comment]
Thanks for the suggestion.
To limit to a certain reputation score would avoid that it can be abused in a graph of fake profiles rating each others.
There would be still some privacy leak as it reveals that both traders have had a trade. Not terrible but something to consider. The user getting rated would have no say if they agree to that decrease of privacy. We could set a flag in settings to allow users to op-out of being rated... but overall positive rating systems come with more problems than negative rating system, e.g. blaming a user who acted bad because such users do not deserve the same protection of privacy as honest users from Bisq's POV. For clear violation we have the ban option of the mediator in place. So I dont think we need something extra here.

We are thinking about some solution also how to get feedback from successful trades in a privacy protecting and not-fakeable way, which is a hard problem, but maybe there is some acceptable compromise for all that...
The core problem is that reputation requires identity and for privacy we don't want to link identity to trade activity. 
The usage of BSQ is a solution to avoid the linkage to identity. Also importing account age and signed account age is done in a way to limit the linkage to the oracle which is a bonded role. The Bisq 2 network does not know about the linkage.
